charts/golang/templates/deployment.yaml

```diff
@@ -88,7 +88,7 @@ spec:
           image: postgres:alpine
           env:
             - name: POSTGRESQL_HOST
-              value: {{ include "golang.postgresqlHost" . }}
+              value: {{ include "golang.postgresqlHost" . | quote }}
```

Comment:
Just out of curiosity, why move the quote out of the template helper and here instead?

Reply:
@GrantSheehan Convention. Say you want to use this value in an output somewhere, by quoting the value in `_helpers` it'll force that value to be quoted wherever you use it. Take the hostname for example, if I quoted in `_helpers`, our `NOTES.txt` would spit out `https://"my-example.local"`.